backend/btrixcloud/crawlmanager.py

```diff
@@ -163,25 +163,45 @@ async def run_re_add_org_pages_job(
             job_id = f"org-pages-{oid}-{secrets.token_hex(5)}"
 
         return await self._run_bg_job_with_ops_classes(
-            oid,
             job_id,
             job_type=BgJobType.READD_ORG_PAGES.value,
+            oid=oid,
             crawl_type=crawl_type,
+            crawl_id=crawl_id,
+        )
+
+    async def run_optimize_pages_job(
+        self, existing_job_id: Optional[str] = None, scale=3
```

Comment:
Should we default this to 1, but allow overriding via helm chart? For default deployment.